apps/src/templates/teacherDashboard/LoginTypePicker.jsx

```diff
@@ -88,6 +89,11 @@ class LoginTypePicker extends Component {
     const withClever =
       providers && providers.includes(OAuthSectionTypes.clever);
     const hasThirdParty = withGoogle | withMicrosoft | withClever;
+    const currentUser = getStore().getState().currentUser;
+    const showStudentsToSectionPermissionWarning =
+      currentUser.isTeacher &&
+      (['US'].includes(currentUser.countryCode) || !!currentUser.usStateCode) &&
```

Comment:
`country_code` is an attribute on the User model which I think is only set for students when we are asking for their us_state. So this won't be helpful for Teacher accounts. To be more consistent, we should look at the detected country in the `request` object: `request.country.to_s.upcase` (or something similar). I see the Mario recently did something similar to pass the value from the backend to the front end: https://github.com/code-dot-org/code-dot-org/pull/57919/files

Reply:
https://github.com/code-dot-org/code-dot-org/pull/58497/commits/98b792955f4ca60e8d3f679e895ebb9b514c96a4